frontend/src/app/shared/components/code-editor/code-editor.component.ts

```diff
@@ -11,6 +18,8 @@ interface EditorOptions {
     styleUrls: ['./code-editor.component.sass'],
 })
 export class CodeEditorComponent implements OnChanges, OnInit {
+    languageNameMap: Map<string, string> = new Map<string, string>([['C#', 'csharp']]);
```

Comment:
I believe there should be something shared

Reply:
with what?

Reply:
I believe we have some shared data for this